python/lsst/fgcmcal/fgcmFitCycle.py

```diff
@@ -757,6 +757,56 @@ class FgcmFitCycleConfig(pipeBase.PipelineTaskConfig,
         default=None,
         optional=True,
     )
+    deltaAperFitMinNgoodObs = pexConfig.Field(
+        doc="Minimum number of good observations to use mean delta-aper values in fits.",
+        dtype=int,
+        default=2,
+    )
+    deltaAperFitPerCcdNx = pexConfig.Field(
+        doc="Number of x bins per ccd when computing delta-aper background offsets.",
+        dtype=int,
+        default=10,
+    )
+    deltaAperFitPerCcdNy = pexConfig.Field(
+        doc="Number of y bins per ccd when computing delta-aper background offsets.",
+        dtype=int,
+        default=10,
+    )
+    deltaAperFitSpatialNside = pexConfig.Field(
+        doc="Healpix nside to compute spatial delta-aper background offset maps.",
+        dtype=int,
+        default=64,
+    )
+    deltaAperInnerRadiusArcsec = pexConfig.Field(
+        doc="Inner radius used to compute deltaMagAper (arcseconds).",
+        dtype=float,
+        default=0.0,
+    )
+    deltaAperOuterRadiusArcsec = pexConfig.Field(
+        doc="Outer radius used to compute deltaMagAper (arcseconds).",
+        dtype=float,
+        default=0.0,
+    )
+    doComputeDeltaAperVisits = pexConfig.Field(
+        doc="Do the computation of delta-aper background offsets per visit?",
+        dtype=bool,
+        default=False,
+    )
+    doComputeDeltaAperStars = pexConfig.Field(
+        doc="Do the computation of delta-aper mean values?",
+        dtype=bool,
+        default=True,
+    )
+    doComputeDeltaAperMap = pexConfig.Field(
+        doc="Do the computation of delta-aper maps?",
```

Comment:
Very good point, I'll add that.  And on further review I realized that it can compute the per-ccd values even if the per-star values weren't computed.  So I've updated the fgcm code, and I'm going to make sure the docstrings are clear here.